algorithmic_efficiency/workloads/criteo1tb/criteo1tb_pytorch/workload.py

```diff
@@ -20,6 +20,10 @@
 
 class Criteo1TbDlrmSmallWorkload(BaseCriteo1TbDlrmSmallWorkload):
 
+  @property
+  def eval_batch_size(self) -> int:
+    return 65536
```

Comment:
Maybe I'm out of the loop here, but is this 1/8th of the `eval_batch_size` for the Jax version of the workload on purpose?

Reply:
yeah, it OOMs if I increase it above this. I assume it has to do with the way we communicate tensors in pytorch, am open to fixes if others can debug it!

Reply:
At what point does it run OOM exactly and does it happen during the first iteration?